### PR TITLE
Fix conflicts and add for loop to prevent duplicates

### PR DESCRIPTION
**Pull request by williamphelps13**
I was able to resolve the very confusing situation we had with github merge conflict. I also added and tested the functionality of saved posters creating duplicates.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -115,8 +115,7 @@ var quotes = [
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
 
-var savedPosters = ["something"];
```
Comment (Samantha-Stevens) [inline src/main.js:118]: Great catch on removing the "something" string from the savedPosters variable.

**Review by Samantha-Stevens — APPROVED** [review]
Yay that Github is up to date and bug free! Great work!
